domain-models-maven-plugin/src/test/java/org/folio/rest/tools/ClientGeneratorTest.java

```diff
@@ -41,7 +41,7 @@ public void doesGenerateTestResourceClient() throws Exception {
     // IDEs always removes trailing spaces from edited files, but java code generator adds then,
     // so we need to remove them before the comparison
 
-    String actual = removeTrailingSpaces(Files.readString(expectedClient.toPath()));
+    String actual = removeTrailingSpaces(Files.readString(expectedClient.toPath()).trim());
```

Comment:
No need to ignore _all_ whitespace.
Simply temporarily disable `@After cleanUp()` and copy the generated
`target/generated-sources/raml-jaxrs/org/folio/rest/client/TestResourceClient.java` to
`src/test/resources/clients/TestClient.txt`.